.eslintrc

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I dogfood'ed the rules in this repo against itself. In order to keep things simple I only  project to the root set of rules and turned off some of them that weren't necessary.